reference/commands/consumer/install.rst

```diff
@@ -216,3 +232,17 @@ With the :command:`-o` parameters you can only define specific package options.
 
     You can use :ref:`profiles <profiles>` files to create predefined sets of **settings**,
     **options** and **environment variables**.
+
+
+reference
+---------
+
+An optional positional argument, if used the first argument should be a path.
```

Comment:
Mention here too that `name` and `version` should match those in recipe if they exist